MANIFEST.in

```diff
@@ -1,18 +1,18 @@
 include MANIFEST.in
 include README.md
 include pyproject.toml
-recursive-include qiskit *pyx
-recursive-include qiskit *pxd
-recursive-include qiskit *.pxi
-recursive-include qiskit *cpp
-recursive-include qiskit *hpp
+recursive-include qiskit_aer *pyx
+recursive-include qiskit_aer *pxd
+recursive-include qiskit_aer *.pxi
```

Comment:
We can probably drop these Cython files?

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/1526/commits/3506bb92ec6312b5090ef24b4718e53a783e41cd